unicode_smilies/unicode_smilies.php

```diff
@@ -1,260 +1,2042 @@
 <?php
 /*
  * Name: Unicode Smilies
- * Description: Smilies based on the unicode emojis - On Linux use https://github.com/eosrei/emojione-color-font to see them in color
+ * Description: Smilies based on the unicode emojis - On Linux use https://github.com/eosrei/emojione-color-font to see them in color and http://www.unicode.org/emoji/charts/full-emoji-list.html
  * Version: 1.0
  * Author: Michael Vogel <http://pirati.ca/profile/heluecht>
  */
 use Friendica\Content\Smilies;
 use Friendica\Core\Hook;
 
 function unicode_smilies_install() {
-	Hook::register('smilie', 'addon/unicode_smilies/unicode_smilies.php', 'unicode_smilies_smilies');
+Hook::register('smilie', 'addon/unicode_smilies/unicode_smilies.php', 'unicode_smilies_smilies');
 }
 
 function unicode_smilies_uninstall() {
-	Hook::unregister('smilie', 'addon/unicode_smilies/unicode_smilies.php', 'unicode_smilies_smilies');
+Hook::unregister('smilie', 'addon/unicode_smilies/unicode_smilies.php', 'unicode_smilies_smilies');
 }
```

Comment:
I think you can remove the `uninstall` function. Right @MrPetovan ?

Reply:
I leave that to those who know about it ;)

Reply:
Absolutely, if it's only `Hook::unregister` calls.